test/kubetest/vpp_utils.go

```diff
@@ -35,22 +35,32 @@ func DeployVppAgentNSC(k8s *K8s, node *v1.Node, name string, timeout time.Durati
 // CheckVppAgentNSC - Perform check of VPP based agent operations.
 func CheckVppAgentNSC(k8s *K8s, nscPodNode *v1.Pod) *NSCCheckInfo {
 	if !k8s.UseIPv6() {
-		return checkVppAgentNSCConfig(k8s, nscPodNode, "172.16.1.1")
+		return checkVppAgentNSCConfig(k8s, nscPodNode, "172.16.1.")
 	}
 	return checkVppAgentNSCConfig(k8s, nscPodNode, "100::1")
 }
 
 func checkVppAgentNSCConfig(k8s *K8s, nscPodNode *v1.Pod, checkIP string) *NSCCheckInfo {
 	info := &NSCCheckInfo{}
-	response, errOut, _ := k8s.Exec(nscPodNode, nscPodNode.Spec.Containers[0].Name, "vppctl", "show int addr")
-	if strings.Contains(response, checkIP) {
+	result := false
+	for i := 0; i < 15; i++ {
```

Comment:
TODO: think how to get rid of this loop. The loop waits for interface crating after recover.